scripts/screenshot.js

```diff
@@ -0,0 +1,9 @@
+const StepRunner = require('../src/step-runner/step-runner');
+const ScreenshotStep = require('../src/step-runner/steps/screenshot/screenshot-step');
+const screenshotStepConfig = require('../src/step-runner/steps/screenshot/screenshot-step-config');
+
+(async () => {
+  await new StepRunner([
+    { step: ScreenshotStep, config: screenshotStepConfig },
+  ]).run();
+})();
```

Comment:
```suggestion
(() => {
  new StepRunner([
    { step: ScreenshotStep, config: screenshotStepConfig },
  ]).run();
})();
```
or maybe even
```suggestion
(() => new StepRunner([
    { step: ScreenshotStep, config: screenshotStepConfig },
  ]).run())();
```
(not sure, too much brightscript on daily basis 😅)

Reply:
changed to:
```js
new StepRunner([
  { step: ScreenshotStep, config: screenshotStepConfig },
]).run();
```